pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -70,12 +70,18 @@ func (c *Cluster) checkHealth() error {
 	c.acquireOrchestrationLock()
 	defer c.releaseOrchestrationLock()
 
-	logger.Debugf("Checking health for mons (desired=%d). %+v", c.spec.Mon.Count, c.clusterInfo)
+	logger.Debugf("Checking health for mons in cluster. %s", c.clusterInfo.Name)
 
 	// Use a local mon count in case the user updates the crd in another goroutine.
 	// We need to complete a health check with a consistent value.
-	desiredMonCount := c.spec.Mon.Count
-	allowMultiplePerNode := c.spec.Mon.AllowMultiplePerNode
+	minMonCount := c.spec.Mon.Count
```

Comment:
Why do you call this "minimum"?

Reply:
These are really the min and preferred target counts. I'll rename them to `minTargetMonCount` and `preferredTargetMonCount`.